lib/zero_phoenix/account/account.ex

```diff
@@ -36,6 +36,31 @@ defmodule ZeroPhoenix.Account do
 
   """
   def get_person!(id), do: Repo.get!(Person, id)
+  def get_person(id), do: Repo.get(Person, id)
+
+  @doc """
+  Gets a list of persons.
+
+  ## Examples
+
+      iex> get_persons!([123])
+      {:ok, [%Person{}]}
+
+      iex> get_persons!([123, 456])
+      {:ok, [%Person{}]}
+
+  """
+  def get_persons(ids) do
+    ids
+    |> Enum.reduce([], fn id, acc ->
+      with person = %Person{} <- get_person(id) do
```

Comment:
This code produces N database queries.  For example, N is equivalent to the number of ids in the given list.